fixtures/legacy-jsx-runtimes/react-15/react-15.test.js

```diff
@@ -0,0 +1,771 @@
+/**
```

Comment:
May want to reference `ReactJSXElement-test` in a comment so it's clear where these come from (and we may want to add a comment in `ReactJSXElement-test` to remind us to update these tests.

Reply:
It’s not just that file but some from a bunch of others too. I could add though.